fabtests/benchmarks/benchmark_shared.c

```diff
@@ -227,7 +227,7 @@ int pingpong(void)
 				return ret;
 		} else {
 			/* Repost RX buffers to support inband sync. */
-			ret = ft_post_rx(ep, rx_size, &rx_ctx);
```

Comment:
Also need a better phrase on the commit message. You need to explain why we need a separate `rx_msg_size`: because it is currently covering both msg and rma operations which may have different max size.